src/export/Export.cpp

```diff
@@ -800,7 +471,6 @@ bool Exporter::ProcessFromTimerRecording(bool selectedOnly,
    mFilename = fnFile;
    mFormat = iFormat;
    mSubFormat = iSubFormat;
-   mFilterIndex = iFilterIndex;
```

Comment:
With this deletion, iFilterIndex becomes unused.

I expected then that Timer record would achieve this side effect otherwise, by a call to configure the exporter, but I don't see it.

Reply:
If that is correct, then `m_iAutoExportFilterIndex` is a useless member of `TimerRecordDialog`.

Reply:
And if that is removed, then `Exporter::GetAutoExportFilterIndex()` will be called nowhere.